packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -248,10 +248,11 @@ export class MultiSelectExample extends React.PureComponent<IExampleProps, IMult
         if (!modifiers.matchesPredicate) {
             return null;
         }
+
         return (
             <MenuItem
-                selected={modifiers.active}
-                icon={this.isFilmSelected(film) ? "tick" : "blank"}
+                active={modifiers.active}
+                selected={this.isFilmSelected(film)}
```

Comment:
~the comment above this method needs to be updated on L246 -- we are not using the default renderer so that we can set `selected` now, not `icon`.~

actually, after removing `renderFilm` from `filmSelectProps` like I suggested above, the comment will no longer be relevant since each component will be implementing its own `renderFilm` method -- the comment on L246 should be removed entirely.